kouta-backend/pom.xml

```diff
@@ -7,11 +7,11 @@
     <parent>
         <groupId>fi.oph.kouta</groupId>
         <artifactId>kouta-backend-parent</artifactId>
-        <version>6.6.0-SNAPSHOT</version>
+        <version>6.7.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>kouta-backend</artifactId>
-    <version>6.6.0-SNAPSHOT</version>
+    <version>6.7.0-SNAPSHOT</version>
```

Comment:
Oikeen version voin kattoo sit ku master merge lähestyy, näitä oli kai tulossa useempi versionnosto